pandas/plotting/_core.py

```diff
@@ -1582,7 +1582,7 @@ def pie(self, **kwargs):
             raise ValueError("pie requires either y column or 'subplots=True'")
         return self(kind="pie", **kwargs)
 
-    def scatter(self, x, y, s=None, c=None, **kwargs):
```

Comment:
you can deprecate but not rename as that is a breaking change

these are matpotlib names but not objecting to the change per se

Reply:
I see your point but I need some help to solve it. What is best practice in this situation and how does a deprication warning work?

Reply:
@mosc9575 check some other PRs which deprecate arguments for examples https://github.com/pandas-dev/pandas/pulls?q=is%3Apr+depr+is%3Amerged+

Reply:
@jreback I used the wrong word. I did not "rename", I added 'size' and `color` to avoid braking code.

Reply:
I wouldn't keep both in the signature, that'd be confusing

I'd suggest:
- use `color` and `size` in the signature
- if `color` is `None`, take `c` from `kwargs` (likewise for `size` and `s`)
- raise an error if someone passes both `color` and `c`
- check that the docs consistently use `color` and `size`

Reply:
@MarcoGorelli Thank you. I go for your suggestions. I will edit my PR.

Reply:
thanks - will also need tests to cover the new behaviour